vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -24,6 +24,8 @@
 
 logger = init_logger(__name__)
 
+_SAFETENSORS_INDEX_FILE_NAME = "model.safetensors.index.json"
```

Comment:
We could possibly reference this directly from transformers https://github.com/huggingface/transformers/blob/4ef85fee718969f1703d7dffa134deb72f4de828/src/transformers/utils/__init__.py#L231

Reply:
done